electron/renderer/components/export-to-language-standalone.jsx

```diff
@@ -10,7 +10,19 @@ class ExportToLanguageStandalone extends Component {
     store: PropTypes.object.isRequired
   };
 
-  handleChange = (event) => {
+  handleFilterChange = (event) => {
+    if (event.keyCode === 13) {
+      try {
+        this.props.appRegistry.emit(
+          'open-query-export-to-language',
+          {filter: event.target.value}
+        );
+      } catch(err) {
+        console.log('Invalid input:' + err.message);
```

Comment:
This is just the skeleton component that opens up the actual components. I think its better to catch the error and log it to the console than more work to make it render the error "prettily", since it's only ever used to test it locally

Reply:
ah gotcha!